docs/api.md

```diff
@@ -1504,6 +1504,33 @@ yargs.showHelp(s => myStream.write(s)); //prints to myStream
 
 Later on, `argv` can be retrieved with `yargs.argv`.
 
+.showVersion([consoleLevel | printCallback])
+---------------------------
+
+Print the version data.
+
+If no argument is provided, version data is printed using `console.error`.
+
+```js
+var yargs = require("yargs")
```

Comment:
Let's show the non-sigleton usage:

```
const yargs = require('yargs/yargs')
const { hideBin } = require('yargs/helpers')
const argv = yargs(hideBin(process.argv)).argv
```

I'm trying to steer people away from using `yargs.foo` directly, because it's more bug prone.